fedimint-core/src/api.rs

```diff
@@ -772,6 +772,21 @@ impl InviteCode {
         ])
     }
 
+    pub fn new_max_guardians_code(
```

Comment:
1. Please document what the fn does
2. Maybe there's a better name, we aren't including the ma number of guardian URLs, but rather the minimum required to always be able to join a working federation